pkg/systemd/init.html

```diff
@@ -0,0 +1,195 @@
+      </ul>
+    </div>
+  </script>
+
+  <script id="service-unit-tmpl" type="x-template/mustache">
+    <div class="panel panel-default">
+      <div class="panel-heading">{{Unit.Description}}</div>
+      <div class="list-group">
+        <div class="list-group-item">
+          <table style="width:100%">
+            <tr>
+              <td style="text-align:left">
+                <span style="font-weight:bold">{{Unit.ActiveState}}</span>
+                ({{Unit.SubState}})
+                <br/>
+                <span translatable="yes">Since {{Since}}</span>
+              </td>
+              <td style="text-align:right">
+                {{{UnitButton}}}
+              </td>
+            </tr>
+          </table>
+        </div>
+        <div class="list-group-item">
+          <table style="width:100%">
+            <tr>
+              <td style="text-align:left">
+                {{#HasLoadError}}
+                {{Unit.LoadState}} (Reason: {{LoadError}})
+                {{/HasLoadError}}
+                {{^HasLoadError}}
+                {{Unit.LoadState}} ({{Unit.FragmentPath}}; {{Unit.UnitFileState}})
+                {{/HasLoadError}}
+              </td>
+              <td style="text-align:right">
+                {{{FileButton}}}
+              </td>
+            </tr>
+            {{#Template}}
+            <tr>
+              <td>This unit is an instance of the <a data-goto-unit="{{Template}}">{{.}}</a> template.</td>
+            </tr>
+            {{/Template}}
+          </table>
+        </div>
+      </div>
+    </div>
+  </script>
+
+  <script id="service-template-tmpl" type="x-template/mustache">
+    <div class="panel panel-default">
+      <div class="panel-heading">{{Id}} Template</div>
+      <div class="list-group">
+        <div class="list-group-item">
+          <input style="width:50em"></input>
+          <button>Instantiate</button>
+        </div>
+      </div>
+    </div>
+  </script>
+
+  <div id="service" class="container-fluid" hidden>
+    <ol class="breadcrumb">
+      <li><a id="service-navigate-home">Services</a></li>
+      <li class="active"></li>
+    </ol>
+
+    <div id="service-valid">
+      <div id="service-unit"></div>
+      <div class="panel panel-default cockpit-log-panel" id="service-log-box">
+        <div class="panel-heading" translatable="yes">Service Journal</div>
+        <div class="panel-body" id="service-log"></div>
+      </div>
+    </div>
+
+    <div id="service-template">
+    </div>
+
+    <div id="service-invalid">
+      <div id="service-error-message"></div>
+    </div>
+
+  </div>
+
+  <div class="modal" id="service-no-install-info-dialog"
+       tabindex="-1" role="dialog"
+       data-backdrop="static">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h4 class="modal-title">Note</h4>
+        </div>
+        <div class="modal-body">
+          <p>This unit is not designed to be enabled explicitly.
```

Comment:
Lots more untranslatable strings that we should probably mark? I won't call them all out individually.